.github/workflows/resolve-pr-release-versions.yml

```diff
@@ -0,0 +1,261 @@
+name: Resolve PR Release Versions
+
+on:
+  workflow_dispatch:
+    inputs:
+      max_release_count:
+        description: Max number (<= 100) of unprocessed releases to process (all if left blank)
+        required: false
+        type: number
+  schedule:
+    - cron: '0 */6 * * *'
+
+permissions:
+  actions: read
+
+jobs:
+  resolve_pr_release_versions:
+    name: Resolve PR Release Versions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Restore previous run data
+        uses: dawidd6/action-download-artifact@71072fbb1229e1317f1a8de6b04206afb461bd67 # v3.1.2
+        with:
+          name: resolved-pr-versions
+          if_no_artifact_found: ignore
+          workflow_conclusion: 'completed'
+          search_artifacts: true
+      - run: npm install @electron/fiddle-core
+      - uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        with:
+          script: |
+            const fs = require('node:fs/promises');
+            const { ElectronVersions } = require('@electron/fiddle-core');
+            const semver = require('semver');
+
+            // https://github.com/electron/trop/blob/a481299dfd522c7b5c5d10e2355ad9e7f0ce193e/src/utils/branch-util.ts#L90-L94
+            const getBackportPattern = () =>
+              /(?:^|\n)(?:manual |manually )?backport (?:of )?(?:#(\d+)|https:\/\/github.com\/.*\/pull\/(\d+))/gim;
+
+            const BOOTSTRAP_DATA_URL = 'https://gist.githubusercontent.com/dsanders11/eb51a04d04a6a3e0710d88db5250e698/raw/fd960b6dea1152b55427407646044f1ba187e52b/data.json';
```

Comment:
I am incredibly nitting here, but do we want to pop this into an Electron repo somewhere? If we think this will actually be pretty temporary, I'm okay leaving it

Reply:
It's a valid point to raise! I'm definitely intending for this workflow overall to be pretty temporary (pending practical realities getting int he way), but this line in particular can be super temporary. Once we merge and have one run of the workflow to store an artifact, this line can be removed entirely as the bootstrap step will be complete and shouldn't (unless artifacts somehow all got jettisoned) be needed again.